internal/server/openapi/error.go

```diff
@@ -15,10 +15,8 @@ import (
 	"net/http"
 )
 
-var (
-	// ErrTypeAssertionError is thrown when type an interface does not match the asserted type
-	ErrTypeAssertionError = errors.New("unable to assert type")
-)
+// ErrTypeAssertionError is thrown when type an interface does not match the asserted type
+var ErrTypeAssertionError = errors.New("unable to assert type")
```

Comment:
I'm a bit perplexed why this generated file has been modified, and allegedly is not re-generated throwing error during GHA for uncommitted modifications; can you elaborate, please?

Reply:
I had previously tried to introduce the new error codes in there, as discussed on last Wednesday's call. Missed the autoformatter change, will remove it.